src/shared/codelens/localLambdaRunner.ts

```diff
@@ -448,7 +448,7 @@ export const invokeLambdaFunction = async (params: {
     baseBuildDir: string,
     channelLogger: ChannelLogger,
     configuration: SettingsConfiguration,
-    debugConfig: DebugConfiguration,
+    debugConfig?: DebugConfiguration,
```

Comment:
This is a great find - looks like we were needlessly passing debugConfig objects when running locally.